README.md

```diff
@@ -1,2 +1,27 @@
-# waku.dev
-Waku.dev
+## How to run the project
+
+1. Install the dependencies:
+
+    ```bash
+    # npm
+    npm install
```

Comment:
Is there a reason we are using npm for waku.dev and yarn for vac.dev? Might be good if we could keep this consistent